cypress/integration/single-planet.js

```diff
@@ -0,0 +1,53 @@
+describe('Single Planet Testing', () => {
+  const baseURL = 'http://localhost:3000/'
+
+  beforeEach(() => {
+    cy.fixture('testPlanets.json')
+      .then(planets => {
+        cy.intercept('https://dry-plains-91502.herokuapp.com/planets', {
```

Comment:
This is intercepting the old API so we'll need to replace it with the URL for the new one.

Reply:
Good catch on this!

Reply:
good catch, fixed!